lib/AddDependencies.js

```diff
@@ -39,6 +40,8 @@ class AddDependencies {
           default: {
             if (!/^-/.test(val)) {
               this.dependencies.push(val);
+            } else if (/package\.json/.test(val)) {
```

Comment:
Probably, you have to use this `if` in the beginning of the `default` block -- to prevent a package.json file path from getting into `this.dependencies` array.

Reply:
already done ;) i found that out the hard way when i tested it haha.